projects/fsastorefrontlib/src/cms-components/checkout/components/quote-review/quote-review.component.ts

```diff
@@ -143,6 +151,26 @@ export class QuoteReviewComponent implements OnInit, OnDestroy {
     }
   }
 
+  private displayQuoteStatusPendingMessage() {
```

Comment:
Change to protected. In this case, If customer wants to change the toast message, he would need to override ngOnInit() and then refer to his implementation. If its protected method, only to override existing displayQuoteStatusPendingMessage.